tutorials/getting-started/setting-up-your-environment.md

```diff
@@ -2,20 +2,15 @@
 title: Setting Up Your Environment
 layout: tutorial
 ---
+# Setting Up Your Environment
 
-# Setting up your environment
+Let's start from the very beginning by installing the software necessary to develop with Apostrophe. After that, we'll set up an empty Apostrophe site.
 
-## Setting up your environment
+## Install the Software
 
-We're going to start from the very beginning by installing the software necessary to develop with Apostrophe. In the next tutorial, we'll set up an empty Apostrophe site.
+>**These instructions are for MacOS X v10.9 or later.** MacOS X is the preferred platform for Apostrophe CMS development. Windows users can use this lovely [HOWTO for Windows](../howtos/windows.md) contributed by Michael Brown to get everything setup. Linux users can get going by installing the same operating system packages required by our [deployment HOWTO](../intermediate/deployment.md).
```

Comment:
setup should be two words in this context yes?

Reply:
I'm concerned that this language makes it sound like developing on Linux is not cool. It is very cool and it is where you will ultimately deploy. If you want to develop on it so much the better, and our apologies for not having a HOWTO just for dev on it.

Reply:
To clarify, I'm worried this wording will be offputting and make people wonder if they can deploy this on Linux or not.

Reply:
Yeah. I think I made this one worse, rather than better. I'll try to make that a little more inclusive to Linux/Windows.

Reply:
Yeah. Windows is a tougher row to hoe, but no point being discouraging.

Reply:
Updated text to:
"MacOS X is the most popular platform for Apostrophe CMS development, but there are plenty of developers using Linux or Windows.  For MacOS 10.9 or newer, continue with the instructions below. Windows users can use this lovely HOWTO for Windows contributed by Michael Brown to get everything set up, and Linux users can get going by installing the same operating system packages required by our deployment HOWTO."